### ISSUE TITLE
Code for training and data generation

### ISSUE THREAD

**Issue by mbpictures** [cong-yang/MoireDet#1]
Hello everyone!

First of all, thanks for your great contribution! There really isn't much out there for moire detection and your work looks very promising. However, I'd like to try your method myself, as I'm currently researching in the area of face liveness detection (aka face anti-spoofing). As I'm not from China, I can't register a Baidu account myself, is there any chance you could (hopefully) upload the datasets to additional cloud storage such as GoogleDrive, Microsoft, Dropbox, etc? Can you maybe share the code you used for training and data generation (combining the moire layers with the real images), as there are many fuzzy variables (e.g. number of filters used in convolutional, etc)?

Looking forward to hearing from you and kind regards,
Marius

**Comment by nataliameira** [comment]
Hey @mbpictures 

Have you trained this repository with your custom dataset?

**Comment by mbpictures** [comment]
@nataliameira yes I actually trained a model with the provided code and dataset. I didn't use a custom dataset and instead managed to download the baidu files somehow (I dont remember how exactly, I did a lot of research in reddit)

**Comment by nataliameira** [comment]
I understand. Thank you for your kind response.

I was unable to download the dataset and I would like to train with my custom dataset.

I didn't understand how to organize the triple dataset. Do you remember what it was like?

**Comment by ajaykrishnan33** [comment]
> [@nataliameira](https://github.com/nataliameira) yes I actually trained a model with the provided code and dataset. I didn't use a custom dataset and instead managed to download the baidu files somehow (I dont remember how exactly, I did a lot of research in reddit)

@mbpictures would you be able to share the dataset with me? Not able to find a way to download from Baidu in India.

**Comment by nataliameira** [comment]
@ajaykrishnan33 Hello, I couldn't get the dataset. Instead, I trained with my own custom dataset. However, I didn't get a significant result. So, after several attempts, I gave up on using this model.

You can follow part of this discussion here: #10
